renew_apple_cert/convert_certificate.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/joho/godotenv"
+)
+
+type inputConv struct {
+	app string
+}
+
+func newInputConv(app string) (i *inputConv) {
+	i = &inputConv{
+		app: app,
+	}
+	return i
+}
+
+func main() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal("Error loading .env file")
+	}
+
+	dirCsr := os.Getenv("DIR_CSR")
+	dirDownloaded := os.Getenv("DIR_DOWNLOADED")
+	dirConverted := os.Getenv("DIR_CONVERTED")
```

Comment:
I think instead of 3 separate directories, we can just have one single directory path, and let the program create the csr, downloaded, and converted sub-directories under it. That way it's more consistent and organized.

Reply:
Agreed.